types/graphql-upload/package.json

```diff
@@ -6,20 +6,20 @@
   },
   "exports": {
     "./package.json": "./package.json",
-    "./GraphQLUpload.js": {
-      "types": "./GraphQLUpload.d.ts"
+    "./GraphQLUpload.mjs": {
```

Comment:
I'm shocked by this, but, the upstream package really _does_ put extensions in its export map. Very odd. Can't be intended...

Reply:
I guess it is. Weird. https://github.com/jaydenseric/graphql-upload/releases/tag/v16.0.0